### DISCUSSION TITLE
Why can't I click on the service tag for log page?

### DISCUSSION THREAD

**Discussion by LuanShiZhiJian** [apache/skywalking#11498] [Q&A]
Skywalking: 8.3

![image](https://github.com/apache/skywalking/assets/41558924/14b2a361-05cb-47f4-b778-30db2bcd9bc4)
![image](https://github.com/apache/skywalking/assets/41558924/4954c799-6694-4471-8904-b5688cf6ac0e)

HOW TO SOLVE IT?THANKS!

**Comment by wu-sheng** [comment]
This version was released 2 years ago. There is ni point to use that.

**Comment by LuanShiZhiJian** [comment]
> This version was released 2 years ago. There is ni point to use that.

Do you have any hints for the crash issue in the new version(9.6,9.5) started?

**Reply by wu-sheng** [reply]
How could I possible know?

I would only say, use a clean database, and check logs.

**Reply by LuanShiZhiJian** [reply]
Thanks, It's my problem, Luck for you!
